angular/src/lib/tooltip/tooltip-container.component.ts

```diff
@@ -37,18 +46,35 @@ export class TooltipContainerComponent implements OnInit {
     this.classList['md-event-overlay--right'] = direction === 'right';
   }
 
+   /** @prop Allows tooltip to stay open when you hover over the tooltip | false  */
+  @Input() allowHover: boolean;
+
   /** @prop shows the arrow or not */
   @Input() showArrow: boolean  = true;
 
   /** @prop Sets the id for screen reader */
   @Input() id: string;
 
-   /** @prop Sets the max-width */
-   @Input() maxWidth: number;
+  /** @prop Sets the max-width */
+  @Input() maxWidth: number;
+
+  @Output() mouseleaveEvent = new EventEmitter();
+  @Output() mouseenterEvent = new EventEmitter();
```

Comment:
should we use camelcase wording mouseLeaveEvent, mouseEnterEvent?

Reply:
I'm ok with not camelCasing these since their browser api counterparts are not camelCased. @bfbiggs?